src/components/content.js

```diff
@@ -0,0 +1,20 @@
+
+import React, { useContext } from 'react'
+import '../assets/sass/content.scss'
+import ipadmini from '../assets/images/ipad-mini.jpg'
+import { ContentContext } from '../containers/App'
+
+const Content = () => {
+  const contentContext = useContext(ContentContext)
+  const currentPageIndex = contentContext.contentState.currentIndex
+  const contentDetails = contentContext?.contentState?.ipadInfos?.content && contentContext.contentState.ipadInfos.content[currentPageIndex]
+
+  return (
+    <div className='content'>
+      {contentContext?.contentState?.ipadInfos?.content && contentContext.contentState.ipadInfos.content[0]?.thumbnail ? <img src={ipadmini} className='photo'/> : null }
+      <div className = 'content-text' dangerouslySetInnerHTML={{ __html: contentDetails && contentDetails?.description }} />
+    </div>
+  )
+}
+
```

Comment:
add propType for all the components.
https://reactjs.org/docs/typechecking-with-proptypes.html